Fizz Buzz/README.markdown

```diff
@@ -55,39 +55,53 @@ Finding numbers divisible by five:
 
 ## The code
 
-Here is a simple implementation in Swift:
+Here is an implementation of the fizzBuzz algorithm using Swift 5 ".isMultiple" function with a Switch statement:
 
 ```swift
-func fizzBuzz(_ numberOfTurns: Int) {
-  for i in 1...numberOfTurns {
-    var result = ""
-
-    if i % 3 == 0 {
-      result += "Fizz"
-    }
-
-    if i % 5 == 0 {
-      result += (result.isEmpty ? "" : " ") + "Buzz"
+(0...50).forEach { (num) in
+    
+    switch num {
+        //Check if 'x' is multiple of 3 and if 'x' is multiple of 5. If yes print fizzBuzz!
+    case let x where x.isMultiple(of: 3) && x.isMultiple(of: 5):
```

Comment:
`isMultiple(of:)` is a good addition, but I'm not sure about having these pattern matching statements.

`if` and `else` statements are a lot more relatable to people. What motivated you to try this approach?